sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/New-SPOSiteManageVersionPolicyJob.md

```diff
@@ -0,0 +1,322 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+---
+
+# New-SPOSiteManageVersionPolicyJob
+
+## SYNOPSIS
+Starts a background job that manages the file versions and version history limits of all of its document libraries.
```

Comment:
Starts a background job to manage file versions and version history limits for all document libraries in the given site.

Reply:
(update other places appropriately)